<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackWidget</class>
 <widget class="QWidget" name="TrackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QVTKWidget" name="visualizationBox" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>351</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>191</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>121</width>
         <height>54</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="lines">
          <property name="text">
           <string>lines</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="tubes">
          <property name="text">
           <string>tubes</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TimeRange</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="totalTimeRange">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>TrackTimeRange</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="begin">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="end">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>GlyphTime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="glyphTime">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="glyphShape">
        <item>
         <property name="text">
          <string>sphere</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cube</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="apply">
        <property name="text">
         <string>Apply parameters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="snapshot">
        <property name="text">
         <string>Take Snapshot</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
